TurkishFilter/sections/specific.txt

```diff
@@ -105,6 +105,10 @@ otonomhaber.com##.mh-content-ad
 tosunkaya.com##.gridlove-sticky-sidebar > div:not([id^="gridlove_posts_widget"])
 kuponla.com##div[class^="kule"]
 kuponla.com##.theiaStickySidebar > #text-18
+liderhaber.com##div[class$="Reklam"]
+liderhaber.com##body > div[style^="text-align: center; width: 980px;"]
+liderhaber.com###siteSag > div[style^="width: 370px; height: auto;"]
+liderhaber.com###siteSol > div > div[style^="text-align: center;width: 728px;"]
```

Comment:
`liderhaber.com###siteSag > div[style^="width: 370px; height: auto;"]` - you have blocked a useful element.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/118768606-e232ae00-b887-11eb-8fb3-c9fc5fe43467.png)

</details><br/>

Where does this apply? `liderhaber.com###siteSol > div > div[style^="text-align: center;width: 728px;"]`

Reply:
@zloyden, http://www.liderhaber.com/kategori/magazin for `liderhaber.com###siteSol > div > div[style^="text-align: center;width: 728px;"]`